src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,21 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private Connection connection;
```

Comment:
Class and Interface Declarations
http://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
With this you mean to change the positions of "Class (static) variables" and Instance variables?

Reply:
Exactly